spring-integration-jdbc/src/test/java/org/springframework/integration/jdbc/StoredProcOutboundGatewayWithSpelIntegrationTests-context.xml

```diff
@@ -57,6 +57,7 @@
 										   stored-procedure-name="GET_MESSAGE"
 										   ignore-column-meta-data="true"
 										   expect-single-result="true"
+										   requires-reply="true"
```

Comment:
Perhaps this should be true by default? That way, there would be no behavior change - exception when the SP returns `null`.